catalogue/aarch64/tests/LB+rel+CAS-no-RtRs-addr.litmus

```diff
@@ -0,0 +1,23 @@
+AArch64 LB+rel+CAS-ok-RtRs-addr
+
+{
+  x = 0;
+  y = 0;
+  z = 2;
+
+  0: X0 = x;      1: X0 = x;
+  0: X1 = y;      1: X1 = y;
+                  1: X2 = z;
+}
+
+P0            | P1                      ;
+
+LDR W3, [X0]  | LDR W3, [X1]            ;
+MOV W4, #1    | EOR W4, W3, W3          ;
+STLR W4, [X1] | CAS W5, W4, [X2]        ;
```

Comment:
is this really a CAS-ok? z is initialised to 2 and CAS compares with 0, or am I missing something?